### DISCUSSION TITLE
How do I send an uploaded pdf file from React application to FastaPI?

### DISCUSSION THREAD

**Discussion by Nathan-Bernardo** [fastapi/fastapi#8739] [Questions]
### Description/Issue
I am trying to send an upload file (e.g.) pdf to FastAPI, but I keep getting the 422 error.  Here is a snippet of the code for both the frontend and backend: https://gist.github.com/Nathan-Bernardo/8e2779dba63f3773ab2a9947c02f89ff.  

Essentially, I'm expecting to just be able to read the file and store it in a directory.  However, I can't do that if I can't get rid of the 422 error.  Any advice?  Here is the error: 
![image](https://user-images.githubusercontent.com/50724423/125559234-86bb60f4-70b7-45b8-bf9e-433550ad246d.png)

### Environment

* OS: Ubuntu 18.04:
* FastAPI Version 0.61.1:

* Python version: 3.8.0

**Comment by Mause** [comment]
Have you defined the `/resume` endpoint as a post endpoint?
